ZotBinsAPI.py

```diff
@@ -7,6 +7,8 @@
 import queries
 import config
```

Comment:
Leaving some comments for fun. They are mostly just nitpick, and I did not manually test anything. Quick glance shows there is no major concern.

Reply:
Thanks for the CR! Appreciate the comments.